### DISCUSSION TITLE
Support reading the config file path from an environment variable.

### DISCUSSION THREAD

**Discussion by IgnisDa** [zitadel/zitadel#5085] [Ideas]
Pretty much what the title says. It gets pretty error prone to have `zitadel --config config/config.yml` littered throughout my scripts. It would be great if there were a `ZITADEL_CONFIG_FILE_PATH` environment variable that would allow you to set this path permanently.

**Comment by fforootd** [comment]
Great idea, we could also us a default path on where we except a file to be by default.

**Reply by IgnisDa** [reply]
If you could guide me to the code where the config file discovery is, I would be glad to make a PR.

**Reply by fforootd** [reply]
If I remember correctly its here https://github.com/zitadel/zitadel/blob/main/cmd/zitadel.go#L65

Right @livio-a ?
